hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/protocolPB/OzoneManagerHARequestHandlerImpl.java

```diff
@@ -70,6 +70,7 @@ public OMResponse handleApplyTransaction(OMRequest omRequest,
     case CreateS3Bucket:
     case DeleteS3Bucket:
     case InitiateMultiPartUpload:
+    case AddAcl:
```

Comment:
Here also we need to have similar logic which is added in OzoneManagerRatisUtils.
As in this patch, only VolumeAddACLRequest is done, rest else should take the default code path (As for these requests, still HA is not in there.)

Reply:
Correct, the scope of this one is just to handle VolumeAddAcl, others will follow once the first one is added.

Reply:
Yes, But when addAcl/removeAcl/setAcl comes for bucket which is not handled, so for them we should call normal handler.handle. So, here we need to have below code
`
if (omClientRequest != null) { OMClientResponse omClientResponse =
          omClientRequest.validateAndUpdateCache(getOzoneManager(),
              transactionLogIndex, ozoneManagerDoubleBuffer::add);
} else {
return handler.handle
}`